src/Workload/Microsoft.Maui.Sdk/Sdk/BundledVersions.in.targets

```
@@ -9,7 +9,7 @@
     <_MauiPlatformName Condition=" '$(TargetPlatformIdentifier)' == 'windows' ">win</_MauiPlatformName>
     <_MauiPlatformName Condition=" '$(TargetPlatformIdentifier)' != 'windows' ">$(TargetPlatformIdentifier.ToLowerInvariant())</_MauiPlatformName>
     <_MauiPlatformName Condition=" '$(_MauiPlatformName)' == '' ">any</_MauiPlatformName>
-    <_MauiRuntimePackAlwaysCopyLocal Condition=" '$(_MauiPlatformName)' == 'any' ">true</_MauiRuntimePackAlwaysCopyLocal>
+    <_MauiRuntimePackAlwaysCopyLocal Condition=" '$(_MauiPlatformName)' == 'any' or '$(SelfContained)' == 'false' ">true</_MauiRuntimePackAlwaysCopyLocal>
```

`$(SelfContained)` can also be blank, so maybe need to put:
```suggestion
    <_MauiRuntimePackAlwaysCopyLocal Condition=" '$(_MauiPlatformName)' == 'any' or '$(SelfContained)' != 'true' ">true</_MauiRuntimePackAlwaysCopyLocal>
```